api/types/src/convert.rs

```diff
@@ -684,10 +684,10 @@ impl<'a, R: MoveResolverExt + ?Sized> MoveConverter<'a, R> {
                             )
                         })
                         .unwrap_or_else(|| {
-                            format!("Move abort: code {} at {}", code, location)
+                            format!("Move abort: code {:#x} at {}", code, location)
```

Comment:
@wrwg this should be a good stopgap until we spend time to improve the error map right?

Reply:
LGTM